test-suite/README.md

```diff
@@ -0,0 +1,130 @@
+# Getting started with Postman tests
+
+## Pre-reqs before running Postman tests
+1. Ensure you have built the docker containers by following the build process on the first README.md
+2. Postman installed
+3. Import the scripts into Postman `nia-patient-switching-standard-adaptor\test-suite\postman\Medicus Test Collection PSS Adaptor.postman_collection.json`
+4. Certificates have been added to postman
+    1. Open Postman / Settings (cog symbol)
+    2. Go to Certificates tab inside Settings menu
+    3. Turn on CA Certificates and add the `rootCA.pem` file from `nia-patient-switching-standard-adaptor\test-suite\postman\localhost-certificates`
+    4. Click on Add certificate next to Client Certificates
+    5. Add the `spineClient` file
+    6. Add the `spintClient.key`
+    7. Change the Host location to `localhost` : `443`
+    8. Click on Add
+    9. Close Settings menu
+
+## Setting up test-suite adaptors to run Postman tests
+1. Open project in IDE
+2. Use the terminal and CD into folder `\nia-patient-switching-standard-adaptor\test-suite`
+3. Run the code `sudo ./start-test-environment.sh`
+4. Allow the system to build adaptors
+5. After building your docker suite should look something like this (containers can appear in any order):
+
+<details>
+    ```
+    - test-suite (expand the folder)
+		- ps_gp2gp_translator-1
+		- gpc_facade-1
+		- mock-spine-mhs-1
+		- inbound-1
+		- ps_db-1
+		- outbound-1
+		- activemq-1
+		- redis-1
+		- dynamodb-1
+    ```
+</details>
+
+## Running the tests (If you are intending run the postman scripts locally then follow these steps)
```

Comment:
I think we should say something like "If you are intending to debug in your IDE" here, as the postman scripts are still run locally in either scenario